lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -169,6 +169,7 @@ export default TestModule.extend({
       Ember.run(function() {
         Ember.setProperties(context, hash);
       });
+      return hash;
```

Comment:
Makes sense to do what the framework does. Not sure I understand the benefit of differentiating on 1.x vs 2.x though.